serverless/pytorch/foolwood/siammask/nuclio/model_handler.py

```diff
@@ -7,24 +7,24 @@
 
 import jsonpickle
 import numpy as np
-from tools.test import *  # pylint: disable=wildcard-import
+import tools.test as impl
 
 
 class ModelHandler:
     def __init__(self):
         # Setup device
-        self.device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
-        torch.backends.cudnn.benchmark = True
+        self.device = impl.torch.device('cuda' if impl.torch.cuda.is_available() else 'cpu')
```

Comment:
`torch` is just PyTorch, so could you import it normally (`import torch`)?

Reply:
I didn't want to change any logic in this script.